.github/workflows/conformance-externalworkloads-v1.10.yaml

```diff
@@ -313,10 +313,17 @@ jobs:
 
       - name: Clean up GKE cluster and VM
         if: ${{ always() }}
-        run: |
-          gcloud container clusters delete ${{ env.clusterName }} --zone ${{ env.zone }} --quiet --async
-          gcloud compute instances delete ${{ env.vmName }} --zone ${{ env.zone }} --quiet
-        shell: bash {0} # Disable default fail-fast behaviour so that all commands run independently
+        uses: nick-invision/retry@3e91a01664abd3c5cd539100d10d33b9c5b68482
+        with:
+          retry_on: error
+          timeout_minutes: 1
+          max_attempts: 10
+          command: |
+            ret0=$(gcloud container clusters delete ${{ env.clusterName }} --zone ${{ env.zone }} --quiet --async) ||
+              echo "$ret0" | tee /dev/stderr | grep "code=404" >/dev/null
```

Comment:
[Can be ignored: it is irrelevant here if we go with the suggestion above, but just so you know:]

If possible, I would recommend using [`--filter`](https://cloud.google.com/sdk/gcloud/reference/topic/filters) and [`--format`](https://cloud.google.com/sdk/gcloud/reference/topic/formats) instead of Bash piping constructs for retrieving information from `gcloud` calls, as it usually is easier to read and maintain afterwards ;)

(and same principle for other cloud providers' CLI calls of course)